runtime/litmus/src/tests/base_call_filter.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright 2020-2022 Litentry Technologies GmbH.
+// This file is part of Litentry.
+//
+// Litentry is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Litentry is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Litentry.  If not, see <https://www.gnu.org/licenses/>.
+
+use super::setup::*;
+use codec::Encode;
+use frame_support::{
+	assert_noop, assert_ok,
+	traits::{VestingSchedule, WrapperKeepOpaque},
+};
+use sp_runtime::traits::Dispatchable;
+type OpaqueCall = WrapperKeepOpaque<<Runtime as frame_system::Config>::Call>;
+
+#[test]
+fn default_mode() {
+	ExtBuilder::default().build().execute_with(|| {
+		assert_eq!(ExtrinsicFilter::mode(), pallet_extrinsic_filter::OperationalMode::Normal);
+	})
+}
+
```

Comment:
I suggest we have an extra test that root call bypasses the filter (just in case of future upstream change)